packages/nextjs/app/builders/0x83DD842BB14aacE72A1a68a129D134Ee3EeBbbd4/page.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import type { NextPage } from "next";
+import { AcademicCapIcon, XMarkIcon } from "@heroicons/react/24/outline";
+import { MapPinIcon } from "@heroicons/react/24/solid";
+
+const Profile: NextPage = () => {
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10 justify-center">
+        <div className="flex gap-10">
+          <div className="">
+            <Image src="/builders.jpeg" className="rounded-full" alt="Sachin Varma" width={200} height={40} />
+          </div>
+          <div className="my-auto flex flex-col gap-1">
+            <div className="text-3xl">Sachin Verma</div>
```

Comment:
This would be a good place to use [semantic HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html), such as an h1.